packages/ui/src/components/Visualization/Canvas/FormV2/fields/ObjectField/GroupFields.tsx

```diff
@@ -0,0 +1,61 @@
+import { FormFieldGroupExpandable, FormFieldGroupHeader } from '@patternfly/react-core';
+import { FunctionComponent, useContext } from 'react';
+import { CanvasFormTabsContext } from '../../../../../../providers';
+import { isDefined } from '../../../../../../utils';
+import { capitalizeString } from '../../../../../../utils/capitalize-string';
+import { SchemaProvider } from '../../providers/SchemaProvider';
+import { ObjectFieldInner } from './ObjectFieldInner';
+import { useFieldValue } from '../../hooks/field-value';
+import { KaotoSchemaDefinition } from '../../../../../../models';
+import { FieldProps } from '../../typings';
+
+interface GroupFieldsProps extends FieldProps {
+  groups: [string, Record<string, KaotoSchemaDefinition['schema']>][];
+  requiredProperties: string[];
+}
+
+export const GroupFields: FunctionComponent<GroupFieldsProps> = ({ propName, groups, requiredProperties }) => {
+  const { selectedTab } = useContext(CanvasFormTabsContext) ?? { selectedTab: 'Required' };
```

Comment:
Perhaps we could repurpose the `CanvasFormTabsContext` to have a default, so we don't need to specify it on every use.

```
export const CanvasFormTabsContext = createContext<CanvasFormTabsContextResult>({
  selectedTab: 'Required',
  onTabChange: () => {},
});
```